src/tactic/abel.lean

```diff
@@ -328,20 +361,24 @@ It attempts to prove the goal outright if there is no `at`
 specifier and the target is an equality, but if this
 fails, it falls back to rewriting all monoid expressions into a normal form.
 If there is an `at` specifier, it rewrites the given target into a normal form.
+`abel!` will use a more aggressive reducibility setting to identify atoms.
```

Comment:
```suggestion

`abel!` will use a more aggressive reducibility setting to identify atoms.
This can prove goals that `abel` cannot, but is more expensive.
```
Just to clarify what direction "more aggressive" points in.